frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -157,6 +157,7 @@ export class StorageClassDropdownInner extends React.Component<
                   ? `${this.props.hideClassName} control-label`
                   : 'control-label'
               }
+              hidden={this.props?.hideLabel}
```

Comment:
This seems like it would break accessibility of this field. What is the purpose of the label change?

Reply:
@spadgett We want to add an asterisk(required field) on the storage class label.

Reply:
Why not add a `required` prop to the storage class dropdown instead of hiding the label and rendering your own? This is a simpler approach and doesn't break accessibility.

Reply:
@spadgett i have tried the required prop, it does not make the label have the asterisk as expected. hence I wanted to make the label hide-able through prop and  render our required label through the <FormGroup />

Reply:
We should fix the bug with the `required` prop so that the label has the asterisk instead.

Reply:
fixed.. should be working as expected now